privacy/whonix.md

```diff
@@ -32,17 +32,17 @@ while the workstation is used for making AppVMs.
 *  [Customizing Whonix](/doc/whonix/customize/)
 *  [Uninstalling Whonix from Qubes](/doc/whonix/uninstall/)
 
-*The following pages are from Whonix's website.*
+*The following pages are written by the Whonix developers and are located on their website.*
```

Comment:
Isn't this inaccurate since Whonix *users* (non-developers) sometimes contribute to the Whonix wiki?

(Note: We're in extreme bikeshedding territory now.)